app/src/main/java/me/tatarka/bindingcollectionadapter/sample/NormalItemViewModel.java

```diff
@@ -0,0 +1,36 @@
+package me.tatarka.bindingcollectionadapter.sample;
+
+import android.view.View;
+
+/**
+ * Created by evan on 6/14/15.
+ */
+public class NormalItemViewModel extends ItemViewModel {
+    public final boolean checkable;
+    private boolean checked;
+
+    public NormalItemViewModel(int index, boolean checkable) {
+        super(index+" a item");
```

Comment:
Should be `"Item " + index`, but why did you remove this from the layout? The header already has a separate layout so it can compute it differently.